pkg/staticpod/staticpod.go

```diff
@@ -143,11 +149,40 @@ func pod(args Args) (*v1.Pod, error) {
 		},
 	}
 
-	if args.CPUMillis > 0 {
-		p.Spec.Containers[0].Resources = v1.ResourceRequirements{
-			Requests: v1.ResourceList{
-				v1.ResourceCPU: *resource.NewMilliQuantity(args.CPUMillis, resource.DecimalSI),
-			},
+	p.Spec.Containers[0].Resources = v1.ResourceRequirements{}
+
+	p.Spec.Containers[0].Resources.Requests = v1.ResourceList{}
+	p.Spec.Containers[0].Resources.Limits = v1.ResourceList{}
+
+	if args.CPURequest != "" {
+		if cpuRequest, err := resource.ParseQuantity(args.CPURequest); err != nil {
```

Comment:
This isn't a blocker to merging but the `if` `else` can be simplified to a single `if`.